processing/src/main/java/org/apache/druid/math/expr/ExprMacroTable.java

```diff
@@ -44,7 +44,8 @@
 public class ExprMacroTable
 {
   private static final List<ExprMacro> BUILT_IN = ImmutableList.of(
-      new BuiltInExprMacros.ComplexDecodeBase64ExprMacro()
+      new BuiltInExprMacros.ComplexDecodeBase64ExprMacro(),
+      new BuiltInExprMacros.StringDecodeBase64UTFExprMacro()
```

Comment:
it doesn't necessarily have to be in this PR, but it would be nice to add an alias for `ComplexDecodeBase64ExprMacro ` (and also the SQL layer), perhaps just by extending the class and overriding the usage of `NAME` so it has consistent naming with this new function, e.g. `decode_base64_complex`

Reply:
I think this can be done registering an AliasedOperatorConversion similar to how SUBSTR or TRUNC is done

Reply:
that would fix the SQL layer, but this is native expression macros, which do not currently have a generic aliasing thing here.

@pranavbhole what are your plans for this aliasing? (I just want to make sure this actually gets done whether or not in this PR, and not lost)

Reply:
created ticket to for aliasing, I will address it in next PR.